### PR TITLE
added FA download icon at the end of toolbar

### LINKED ISSUE #142: Add Download Icon
image

- Use the FaDownload icon
- Put the icon at the end of the toolbar component (end of the search/reset)
- Does not need functionality
- These changes should automatically affect participants, teams, judges, volunteers, mentors

### PR DESCRIPTION
**Pull request by fardinzam**
Screenshot 2023-07-29 at 2 06 26 PM
Screenshot 2023-07-29 at 2 06 37 PM
Screenshot 2023-07-29 at 2 06 49 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
   },
   "dependencies": {
     "@auth/firebase-adapter": "^1.0.0",
+    "@fortawesome/fontawesome-svg-core": "^6.4.0",
```
Comment (menthy-wu) [inline package.json]: Don't install these packages, use react icons, https://react-icons.github.io/react-icons/icons?name=fa

File: src/components/Toolbar.jsx [hunk 2]
```diff
@@ -3,6 +3,8 @@ import React, { useState } from "react";
 import Checkbox from "./Checkbox";
 import { HiSearch } from "react-icons/hi";
 import Tag from "./Tag.jsx";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```
Comment (menthy-wu) [inline src/components/Toolbar.jsx]: use the fontawesome in react icons

**Review by menthy-wu — CHANGES_REQUESTED** [review]

File: src/components/Toolbar.jsx (hunk 2 above)
Comment (fardinzam) [reply src/components/Toolbar.jsx]: removed that line

File: package.json (hunk 1 above)
Comment (fardinzam) [reply package.json]: took that line out, used react icon

**Review by shahdivyank — APPROVED** [review]

**Review by menthy-wu — APPROVED** [review]

**Pull request merged**